include/UnicycleNavigationController.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 20023 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Simone Micheletti
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ */
+
+#ifndef UNICYCLE_NAVIGATION_CONTROLLER_H
+#define UNICYCLE_NAVIGATION_CONTROLLER_H
+//Fix for compiling math constants
+#define _USE_MATH_DEFINES
+#include <math.h>
```

Comment:
I think you could remove this. In the cpp you used ``cmath`` that is better, since it adds the functions like ``pow``, ``sin`` and ``cos`` to the ``std`` namespace

Reply:
I have removed the header, but on internet I found [this](https://stackoverflow.com/questions/6563810/m-pi-works-with-math-h-but-not-with-cmath-in-visual-studio) that states that the include guard should be kept. I it is old, however

Reply:
I was mentioning to remove only 
```
#include <math.h>
```

GitHub visualizes also the two lines above. I definitely agree in keeping
```
#define UNICYCLE_NAVIGATION_CONTROLLER_H
```